src/Sylius/Behat/Page/Shop/Error/ErrorPage.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Shop\Error;
+
+use Sylius\Behat\Page\Page;
+
+class ErrorPage extends Page implements ErrorPageInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getUrl(array $urlParameters = [])
```

Comment:
missing return type

Reply:
It's an abstract on Sylius\Behat\Page
`abstract protected function getUrl(array $urlParameters = [])`
But this is a protected method, I need to change that